debezium-core/src/main/java/io/debezium/pipeline/ChangeEventSourceCoordinator.java

```diff
@@ -146,6 +126,39 @@ public synchronized void start(CdcSourceTaskContext taskContext, ChangeEventQueu
         }
     }
 
+    protected void doSnapshot(CdcSourceTaskContext taskContext, SnapshotChangeEventSource<P, O> snapshotSource, Offsets<P, O> previousOffsets,
```

Comment:
One more remark, the method is called `doSnapshot` but in fact it does the snapshot and then switch to the streaming. So either the name should reflect it or the code should be slightly refactored and have two methods `doSnapshot` and `doStreaming`.

Reply:
I've renamed the method to `executeChangeEventSources.